app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="jumbotron mt-4 alert alert-light offset-md-2 col-md-8 offset-md-2">
+
+<h1 class="text-center mt-4">Hello everyone!!</h1>
+<div class="row justify-content-around mt-4">
+  <div class="card border-danger mb-3 col-4" style="max-width: 20rem;">
+    <div class="card-header"><strong>About This Site</strong></div>
+    <div class="card-body text-danger">
+      <h5 class="card-title">e-learning</h5>
+      <p class="card-text">You can learn English with this site.</p>
+      <p>It can be used for Eiken and TOEIC test preparation. </p>
+      <p>You can also connect with your friends and see each other's progress. Let's study while encouraging each other!</p>
+      <p>Have a good time.</p>
+      <img src="home_imag.jpg" width="220" height="150">
```

Comment:
please change this code to the following:
`<%= image_tag ("home_imag.jpg", width: 220, height: 150) %>`